### ISSUE TITLE
Build optimizations

### ISSUE THREAD

**Issue by ymoog** [phytec/doc-bsp-yocto#147]
1. Sphinx doc section of the[Latex builder](https://www.sphinx-doc.org/en/master/usage/builders/index.html#sphinx.builders.latex.LaTeXBuilder)  highlights the ability to silence the build:
```bash
make latexpdf LATEXMKOPTS="-silent"
```
and to pass specific latex options.

2.  In addition, we might be able to parallelize the pdf build to speed it up? Although I am not sure if this is possible. 
3. Use [-j](https://www.sphinx-doc.org/en/master/man/sphinx-build.html#cmdoption-sphinx-build-j) to speed up build targets

**Comment by jremmert-phytec-iot** [comment]
Good point! We can let sphinx create only the latex files, but not compile them. Once the latex files exist, we can build them in parallel with make. I can open a PR.

**Comment by ymoog** [comment]
Do you also want to use the ``--jobs auto`` option for ``sphinx-build``?
